### ISSUE TITLE
Build the aurelia-community.space website

### ISSUE THREAD

**Issue by adriatic** [aurelia-community/aurelia-community.github.io#1]
## Introduction
This website serves the same role and provides similar UX as the original **[Aurelia website](https://aurelia.io)**

**Comment by adriatic** [comment]
###  `Guides`

**Comment by adriatic** [comment]
## `Blog`

**Comment by adriatic** [comment]
@RomkeVdMeulen you continue to imporess me by pointing out places where I was not careful enough. I do really appreciate this, so can we establish a more official relationship in the context of AUCS?

**Comment by RomkeVdMeulen** [comment]
I do like this project and what it's trying to achieve, and I like writing. I can't make any guarantees on promptness, since the time I have available to work on OSS varies, but if that's acceptable I'd love to help out.

**Comment by adriatic** [comment]
I do not believe that OSS projects depend on or require any guarantees on promptness Romke 😄, so welcome to the team:

![image](https://user-images.githubusercontent.com/2712405/36284016-3f6509a0-1274-11e8-9761-928185d77635.png)

If you are not a member of the Aurelia's discourse forum, please go there and look for AUCS article.

Please start by reading **[this article](https://github.com/aurelia-community/AUCS-projects-wide-collaboration/issues/2#issue-287176724)** first
